bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/listings/ListingsTableAccessor.java

```diff
@@ -63,15 +63,49 @@ public List<Listing> getAllListings() throws SQLException {
 		return listings;
 	}
 
-	public Listing getListingById(int listingId) {
-		// TODO
+	public Listing getListingById(int listingId) throws SQLException {
+		String sql = "SELECT * FROM listings WHERE listing_Id = ?",
```

Comment:
`listing_Id` should be `listing_id`, you should always double check with the DB models to make sure your table and column names match up with the Java code